### PR TITLE
Update index-africa

### PR DESCRIPTION
**Pull request by josoga2**

### CODE REVIEW

**Comment by bgruening** [comment]
You need to merge master into your branch. I changed the filename for you.

File: index-africa [hunk 1]
```diff
@@ -4,27 +4,47 @@ website: https://africa.usegalaxy.eu
 subdomain: Africa
 ---
 
-# Welcome to africa.usegalaxy.eu, the African galaxy server
+![African Galaxy](<img src="https://avatars0.githubusercontent.com/u/68950141?s=200&v=4" height="100px" alt="ASaiM logo"/>){:.sc-intro-left}
```
Comment (bgruening) [inline index-africa:7]: "ASaiM logo" should be fixed.

**Comment by bgruening** [comment]
@josoga2 there is still a merge conflict :(

File: index-africa [hunk 2]
```diff
@@ -4,27 +4,47 @@ website: https://africa.usegalaxy.eu
...
+  * 16S metagenomics
+  * Metatranscriptomics
+Machine learning
+  * Classification & Regression
+  * Clustering
+  * Regression
+  * Deep learning
+  
+# Appreciation
+We sincerely appreciate the efforts of the following individuals and their organizations for supporting the African Galaxy Instance:
```
Comment (beatrizserrano) [inline index-africa:50]:
```suggestion
We sincerely appreciate the efforts of the following individuals and their organizations for supporting the African Galaxy Instance.
```

File: index-africa (hunk 1 above)
Comment (josoga2) [reply index-africa:7]: Yeah! I needed to move home... So I decided to keep the commits... I'm on it... I have not completed the edits

**Pull request closed without merging**